examples/Calc/CMakeLists.txt

```diff
@@ -1,16 +1,14 @@
 project(Calc)
 
-include_directories(${CUKE_INCLUDE_DIRS} src)
-
 add_library(Calc src/Calculator)
+target_include_directories(Calc PUBLIC src)
```

Comment:
For future reference: I would have preferred for this cleanup to happen in a separate PR, or at least a separate commit. Generally I prefer every change in one commit to work towards accomplishing one goal (which would be fixing the build for this one).